test/plugin.test.js

```diff
@@ -0,0 +1,464 @@
+'use strict'
+
+const MachinomyServerPlugin = require('..')
+const HDWalletProvider = require('truffle-hdwallet-provider')
+const Store = require('./util/memStore')
+const BtpPacket = require('btp-packet')
+const IlpPacket = require('ilp-packet')
+const debug = require('debug')('ilp-plugin-eohereum-asym-server:test')
+const getPort = require('get-port')
+const formatAmount = require('../build/index.js').formatAmount
+const crypto = require('crypto')
+
+const secret = 'lazy glass net matter square melt fun diary network bean play deer'
+const providerUrl = "https://ropsten.infura.io/T1S8a0bkyrGD7jxJBgeH"
+const provider = new HDWalletProvider(secret, providerUrl, 0)
+function sha256 (preimage) { return crypto.createHash('sha256').update(preimage).digest() }
+
+class Trace extends Error {}
+
+/* TODO Mock for web3, HDWallet, and Machinomy API. */
+async function createPlugin() {
+  /* Silence console log output from provider creation. */
```

Comment:
Can you make an upstream PR to use debug instead of console.log? It's quite strange for a library to use console.log without an option to turn it off

Reply:
Done.

Reply:
Submitted.